app/views/_flash_msg.html.erb

```diff
@@ -2,7 +2,7 @@
   <% if flash[type].present? %>
     <div class="alert <%= flash_dom_class %> alert-dismissable" role="alert">
       <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-      <%= safe_join(Array.wrap(flash[type]), '<br/>') %>
+      <%= safe_join(Array.wrap(flash[type]).map(&:html_safe), '<br/>'.html_safe) %>
```

Comment:
@jcoyne You can see the commit that pulled in `safe_join` removed explicit calls to `#html_safe`: https://github.com/projecthydra-labs/hyrax/commit/be4a648f9e0f9191f514a241780dabab29c73777 So I believe we still need it.